backend/posts/posts.py

```diff
@@ -1,23 +1,42 @@
 """Functions related to post management and database communcation."""
 
 from typing import List
-from firebase_admin import firestore, initialize_app
+
+import firebase_admin   
+from firebase_admin import firestore, initialize_app, credentials, storage
+import pyrebase
+
+from flask import Flask, render_template, request, flash, redirect, url_for
+from werkzeug.utils import secure_filename
+
 from google.cloud.exceptions import NotFound
 from intelligence import generate_image_metadata
 from models import AddPostMetadata, PostMetadata
 from exceptions import CreatePostException, GetPostException, UnknownPostIdException, DeletePostException
 
 COLLECTION_POSTS = u'/posts'
 
+
+firebaseConfig = {
+  "apiKey": "",
+  "authDomain": "hashpost.firebaseapp.com",
+  "databaseURL": "https://hashpost.firebaseio.com",
+  "projectId": "hashpost",
+  "storageBucket": "hashpost.appspot.com",
+  "messagingSenderId": "142555671587",
+  "appId": "1:142555671587:web:6807408877be551d391596"
+}
+
 # Initialize Firebase with Application Default Credentials
 initialize_app(name='hashpost')
-
 db = firestore.client()
 
+firebase = pyrebase.initialize_app(firebaseConfig) # Intialize Firebase with configuration settings
```

Comment:
I didn't notice this the first time around, but we're going to have to use a different means (not pyrebase, which is unmaintained) to access Cloud Storage.